webpack.config.dev.js

```diff
@@ -0,0 +1,47 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
```

Comment:
Not an error, but you can try to use https://www.npmjs.com/package/webpack-merge to reduce some code repeating

Reply:
Thank you for the advice!